test/jit/test_cuda.py

```diff
@@ -86,36 +86,56 @@ def event_default_args() -> bool:
             return e is not None
 
         @torch.jit.script
-        def event_default_args_1() -> bool:
+        def event_arg_blocking() -> bool:
             e = torch.jit.cuda.Event(blocking=True)
-            return e is not None
+            s0 = torch.cuda.current_stream(0)
+            s = torch.jit.cuda.Stream()
+            e.record(s0)
+            e.wait(s)
+            A = torch.rand(1000, 1000, device="cuda")
+            with torch.jit.cuda.stream(s):
+                B = torch.mm(A, A)
+            s.synchronize()
+            e.record(s0)
+
+            return s.query() and e.query()
 
         @torch.jit.script
-        def event_default_args_2() -> bool:
-            e = torch.jit.cuda.Event(enable_timing=True)
-            return e is not None
+        def event_arg_enable_timing() -> float:
+            e_tik = torch.jit.cuda.Event(enable_timing=True)
+            e_tok = torch.jit.cuda.Event(enable_timing=True)
+            s0 = torch.cuda.current_stream(0)
+            e_tik.record(s0)
+            A = torch.rand(1000, 1000, device="cuda")
+            with torch.jit.cuda.stream(s0):
+                B = torch.mm(A, A)
+            s0.synchronize()
+            e_tok.record(s0)
+            e_tok.synchronize()
+            return e_tik.elapsed_time(e_tok)
```

Comment:
It's difficult to prove that they are non-blocking, because of `sleep` not being available. So I instead verified if `enable_timing=True` can be verified.